eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -62,9 +68,28 @@ export const StatelessBasicAnswer = ({
   optionErrorMsg,
   multipleAnswers,
 }) => {
-  let [toggled, setToggled] = useState(false);
+  const [toggled, setToggled] = useState(false);
+  const [mutuallyExclusiveLabel, setMutuallyExclusiveLabel] = useState();
+
+  const [createMutuallyExclusiveOption] = useMutation(
+    CREATE_MUTUALLY_EXCLUSIVE_OPTION
+  );
+  const [updateOption] = useMutation(UPDATE_OPTION_MUTATION);
+  const [deleteOption] = useMutation(DELETE_OPTION);
+
   const errorMsg = buildLabelError(MISSING_LABEL, `${lowerCase(type)}`, 8, 7);
 
+  useEffect(() => {
+    const { options } = answer;
```

Comment:
const { options } = answer;
little curious why this is not global  as it's defined in 3 places?

Reply:
Honestly, there's no reason. I can take a look at making it global if you'd like? I'd be tempted to do something like this:

```
const [options, setOptions] = useState({});

useEffect(() => {
   setOptions(answer.options);
}, [answer])

useEffect(() => {
   const mutuallyExclusive = options?.find(
      ({ mutuallyExclusive }) => mutuallyExclusive === true
    );
    if (mutuallyExclusive) {
      setToggled(true);
      setMutuallyExclusiveLabel(mutuallyExclusive.label);
    }
}, [options])

. . .
```
The reason I'd do it like this is to ensure that things update and run only when the props update; they all depend on `answer` changing. At least, most of them do.

Reply:
no worries - looks more simple to declare locally!